beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  title: string;
+  gameTime: string;
+  phase: string;
+  season: string;
+  year: string;
+  gameType: string;
+  playType: string;
```

Comment:
what's play type?

Reply:
Ive changed the playType to GameMode. with is easier to understand.  Bot/multiplayer